src/SDKs/DataMigration/DataMigration.Tests/DataMigration.Tests.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@hitenjava model your project looking at other projects in the repo (e.g. https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/BatchAI.Tests/BatchAI.Tests.csproj

Reply:
Done.